src/marp.ts

```diff
@@ -13,7 +13,13 @@ import uncoverTheme from '../themes/uncover.scss'
 
 export interface MarpOptions extends MarpitOptions {
   emoji?: emojiPlugin.EmojiOptions
-  html?: boolean | { [tag: string]: string[] }
+  html?:
+    | boolean
+    | {
+        [tag: string]:
+          | string[]
+          | { [attr: string]: boolean | ((value: string) => string) }
```

Comment:
We recognize that Marp Core has several risks by disabled `js-xss`'s default attribute sanitizer on the change of this PR. So we updated type definition of `html` option to allow custom attribute sanitizer by passing function.